rex-release/out.sh

```diff
@@ -47,6 +47,22 @@ upload-release() {
 
   # books files with explicit content type set because they don't have extensions, loaded unversioned so no cloudfront caching
   aws s3 sync --exclude 'service-worker.js' --content-type 'text/html' --cache-control 'max-age=0' "$path/books/" "s3://$bucket/rex/releases/$version/books"
+
+  # configure redirects
+  for row in $(cat $path/rex/redirects.json | jq -r -c '.[]'); do
```

Comment:
linter is gonna complain about use of cat here, i think the right way is `jq -c '.[]' < "$path"/rex/redirects.json`

i also don't think you want that `-r`